src/ComplexTypes/ArrayOfTimeframeTimeFrame.php

```diff
@@ -0,0 +1,41 @@
+<?php namespace DivideBV\Postnl\ComplexTypes;
+
+class ArrayOfTimeframeTimeFrame extends BaseArrayOfType
+{
+
+    /**
+     * The name of the array property this class is a wrapper of.
+     */
+    const WRAPPED_PROPERTY = 'TimeframeTimeframe';
+
+    /**
+     * @var TimeframeTimeFrame[] $TimeframeTimeFrame
+     */
+    protected $TimeframeTimeFrame = null;
+
+    /**
+     * @param TimeframeTimeFrame[] $TimeframeTimeframe
+     */
+    public function __construct(array $TimeframeTimeframe)
+    {
+        $this->setTimeframeTimeFrame($TimeframeTimeframe);
+    }
+
+    /**
+     * @return TimeframeTimeFrame[]
+     */
+    public function getTimeframeTimeFrame()
+    {
+        return $this->TimeframeTimeFrame;
+    }
+
+    /**
+     * @param TimeframeTimeFrame[] $TimeframeTimeFrame
+     * @return ArrayOfTimeframeTimeFrame
+     */
+    public function setTimeframeTimeFrame(array $TimeframeTimeFrame)
+    {
+        $this->TimeframeTimeFrame = $TimeframeTimeFrame;
```

Comment:
Yeah, that's actually how they are called in CIF. Basically just used a WSDL to PHP library to get most of these types. O_O;